server/src/db/model.ts

```diff
@@ -21,6 +30,17 @@ const userSchema = new mongoose.Schema({
         height: { type: Number, required: true },
         fitnessLevel: { type: String, required: true },
     },
+    //using this field if the user is given the choice to skip the inetrest form
+    seen_greeting_modal: { type: Boolean, default: false },
```

Comment:
not sure if this field is needed since the user will only see the form once and they can skip it and cant access it anymore, but it doesn't break anything so its fine either way